reference/config_files/source_credentials.rst

```diff
@@ -0,0 +1,81 @@
+.. _reference_config_files_source_credentials:
+
+source_credentials.json
+=======================
+
+.. include:: ../../common/experimental_warning.inc
+
+
+When a ``conanfile.py`` recipe downloads some sources from other servers with the ``download()`` or the ``get()`` helpers like:
+
+.. code-block:: python
+
+    def source(self):
+        # Immutable source .zip
+        download(self, f"https://some/server/tarballname-{self.version}.zip", "downloaded.zip")
```

Comment:
Using the same URL thru the file to show that you won't need to match the whole URL, just the begining

```suggestion
        download(self, f"https://server/that/need/credentials/files/tarballname-{self.version}.zip", "downloaded.zip")
```